### ISSUE TITLE
Request <-> Collection map

### ISSUE THREAD

**Issue by stofte** [postmanlabs/postman-docs#1461]
Is there anyway to see what collection a request I have open in PostMan belongs to? Should I give the request a unique name, and then look for it in my collections?

**Comment by ArifPMan** [comment]
Apologies as this got buried in the stack. Do you think this is still relevant? Let me know so I can get it directed to an appropriate person.

**Comment by stofte** [comment]
I definently think it's relevant. My usual workflow with postman usually involves opening PM and seeing dozens of open tabs, which I proceed to close with and ignoring "Save changes?" because it's really not easy to correlate.

Tabs I cloned, I dont mind if I losee, but would be nice to be able to actively decide for requests that are part of a collection, but it's really not obvious to me how to correlate the two views. From a bit of experimentation I can expand all my collections, and it seems like it should be highlighted in the collection.

Maybe the tabs could be colorized? Generally I have to use PM because it's the most convinient tool for HTTP that I know of, but I generally find the UX to be quite confusing. My own view is that the product is not for me as a developer, but more oriented towards large test teams and the like, instead of easily dealing with ad-hoc queries and adjustsments. SoapUI was very similar in this regard, in that ad-hoc flows were not the main usecases.

Given that collections highlight if its expanded, there's a workaround I can utilize. Closing from my viewpoint.
